releng/third-party/pom.xml

```diff
@@ -45,8 +46,10 @@
 		<hdrhistogram.version>2.1.12</hdrhistogram.version>
 		<jakarta.activation.version>1.2.1</jakarta.activation.version>
 		<jakarta.mail.version>1.6.5</jakarta.mail.version>
+		<javax.websocket.version>1.1</javax.websocket.version>
+		<jetty.version>9.4.43.v20210629</jetty.version>
```

Comment:
I have seen in the PR description this codes needs to stick with 9.4.43…. But I noticed the coordinate of the artefacts may have changed after version 9 (group:`org.eclipse.jetty.websocket`, artefact: `wesocket-server` -> `websocket-jetty-server`). I haven't yet checked the dependencies in the POM, maybe that helps.

By the way there's a more recent version of [jetty 11.0.6](https://search.maven.org/artifact/org.eclipse.jetty.websocket/websocket-jetty-server/11.0.6/jar).

Reply:
Interesting, thanks for pointing this out!

I picked 9.4.43 because it was the latest version I saw here https://mvnrepository.com/artifact/org.eclipse.jetty.websocket/websocket-server I didn't notice the rename for later versions. 😭 

Maybe I can get rid of the dependency issues if I just use `10.0.5` which is already required by _something_ (I don't know what) in the uitests.

Reply:
So it seems you finally go this working :)